docs/contributing.rst

```diff
@@ -267,6 +267,17 @@ Please:
 .. _PEP 8 - Style Guide for Python Code:
   https://www.python.org/dev/peps/pep-0008
 
+Submitting a pull request
+=========================
+Steps:
```

Comment:
Missing empty line between section header and contents. This is not only a nit, but something dictated by rst format and Sphinx AFAIR.